examples/static/samples/files/amp-form-success.html

```diff
@@ -0,0 +1,25 @@
+<!---
```

Comment:
nit: you can remove all the comments as they're no longer needed

Reply:
PTAL